resources/js/Pages/Top.jsx

```
@@ -0,0 +1,25 @@
+import { Head } from '@inertiajs/react'
+import { SideMenu } from '../Components/SideMenu'
+import { LogoutButton } from '../Components/LogoutButton'
+
+export default function Top({}) {
```

Biome (1.9.4)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)